blockchain/blockchain.go

```diff
@@ -456,23 +456,32 @@ func stateUpdateByHash(txn db.Transaction, hash *felt.Felt) (*core.StateUpdate,
 	})
 }
 
+type OnUnknownClass func(classHash *felt.Felt) (core.Class, error)
```

Comment:
Although this function type allows us not to import the feeder package however I don't think it is required. I think we can still do the fetching in the sync package.